packages/gasket-utils/test/require-with-install.test.js

```diff
@@ -152,4 +152,17 @@ describe('requireWithInstall', function () {
         .equals(true);
     });
   });
+
+  describe('list of packages', function () {
+    it('allows for the first parameter to be an array of strings', async function () {
```

Comment:
Could you include in this test (or a separate one) a check that the packageManager (using `packageManagerExecStub`) is called with both packages for the install command?